### PR TITLE
Add script to get S3 bucket size.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: tools/aws/s3_bucket_size.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Queries Cloud Watch to get the size of each S3 bucket in the account.
+#
+set -euo pipefail
+
+YESTERDAY=$(date +%Y-%m-%d --date=yesterday)
```
Comment (duallain) [inline tools/aws/s3_bucket_size.sh:7]: Had some trouble with this (for whatever reason) on my mac.  Does this work on your machine as well?

```YESTERDAY=$(date -v -1d '+%Y-%m-%d')```

Comment (glogiotatidis) [reply tools/aws/s3_bucket_size.sh:7]: No doesn't work for me. I'm using GNU coreutils `date`.

Comment (duallain) [reply tools/aws/s3_bucket_size.sh:7]: what do you think about something like this:

```if [ $( uname -s ) == "Darwin" ]
then
    YESTERDAY=$( date -v -1d '+%Y-%m-%d' )
else
    YESTERDAY=$( date +%Y-%m-%d --date=yesterday )
fi```

Comment (glogiotatidis) [reply tools/aws/s3_bucket_size.sh]: updated

**Pull request merged**